lib/column/tuple.go

```diff
@@ -566,6 +570,23 @@ func (col *Tuple) AppendRow(v any) error {
 		return nil
 	}
 
+	if tuple2, ok := v.(Tuple2); ok {
```

Comment:
`switch valuer := v.(type)` since we're checking between Tuple2/driver.Valuer/default